File: Assets/Globals.cs
using UnityEngine;
using System.Collections;
using Noam_Library;
    public class Globals : MonoBehaviour {


        //that script should be attached to the father of all the tranforms

        public static Hashtable transformTable;
        public void Start() {
            ArrayList allChildren = Library.AllChildren(transform);
            transformTable = new Hashtable();

            foreach (Transform child in allChildren) {
                transformTable.Add(child.name, child);

            }

        }





        static Transform getFromTable(string key) {
            return transformTable[key] as Transform;

        }

        public void Update() {


        }
    }

File: Trash/Globals.js
#pragma strict

//that script should be attached to the father of all the tranforms
static var transformTable : Hashtable;
function Start () {
	var allChildren : Array = Library.AllChildren(transform); 
	transformTable = new Hashtable();
	for (var child : Transform in allChildren){
		transformTable.Add(child.name, child);
	} 
}


static function getFromTable(key : String) : Transform{
	return transformTable[key] as Transform;
}
function Update () {

}